docs/guides/advanced-cluster-new-sharding-schema.md

```diff
@@ -1,24 +1,28 @@
 ---
-page_title: "advanced_cluster - Migration to new sharding schema and leveraging Independent Shard Scaling"
+page_title: "Migration Guide: Advanced Cluster New Sharding Schema"
 ---
 
+# Migration Guide: Advanced Cluster New Sharding Schema
+
 **Objective**: Guide users to migrate their existing advanced_cluster configurations to use the new sharding schema which was introduced in version `1.18.0`. Additionally, a section is included to describe how Independent Shard Scaling can be used once the new sharding schema is adopted. Exiting sharding configurations will continue to work but will have deprecation messages if not using the new sharding schema.
```

Comment:
Rewrite as: Use this guide to migrate your existing `advanced_cluster` configurations to the new sharding schema introduced in version `1.18.0`. The new sharding schema allows you to scale shards independently. Exiting sharding configurations continue to work, although the software issues deprecation messages if you use the legacy schema.